modules/services.js

```diff
@@ -0,0 +1,121 @@
+const https = require('https')
+const axios = require('axios').create({
+  httpsAgent: new https.Agent({
+    rejectUnauthorized: process.env === 'production' || false
+  })
+})
+
+const { witnessMessage, pAndCMessage } = require('./slack/messages')
+const { getSlackUserProfile, sendSlackMessage } = require('./slack/web_client')
+const { logServiceError } = require('./utils')
+const { formatUserData } = require('./utils')
+
+const { API_URL, PNC_CHANNELS } = process.env
+
+/**
+ * Notify Specified People And Culture Channels
+ *
+ * @param {Object} payload the incident created on the wire api
+ *
+ * @returns {Promise} a promise containing the notification response
+ */
+async function notifyPAndChannels (payload) {
+  try {
+    const { reporter: [{ location: { country } }] } = payload
+    const channel = PNC_CHANNELS.split(',').find(
+      value => value.toLowerCase().includes(country.toLowerCase())
+    ).replace(`-${country.toLowerCase()}`, '')
```

Comment:
Not sure of the current slack names for these channels. Will have to confirm this when we have the first production deployment.

Reply:
sure, as a rule they should conform to prefixing it with country names if they can help it.